src/LazyImage/src/Twig/BlurHashExtension.php

```diff
@@ -22,28 +21,9 @@
  */
 class BlurHashExtension extends AbstractExtension
 {
-    private $blurHash;
-
-    public function __construct(BlurHashInterface $blurHash)
-    {
-        $this->blurHash = $blurHash;
-    }
-
-    public function getFunctions(): array
-    {
-        return [
-            new TwigFunction('data_uri_thumbnail', [$this, 'createDataUriThumbnail']),
-            new TwigFunction('blur_hash', [$this, 'blurHash']),
-        ];
-    }
-
-    public function createDataUriThumbnail(string $filename, int $width, int $height, int $encodingWidth = 75, int $encodingHeight = 75): string
-    {
-        return $this->blurHash->createDataUriThumbnail($filename, $width, $height, $encodingWidth, $encodingHeight);
-    }
-
-    public function blurHash(string $filename, int $encodingWidth = 75, int $encodingHeight = 75): string
+    public function getFunctions(): iterable
```

Comment:
```suggestion
    /**
    * @return TwigFunction[]
    */ 
    public function getFunctions(): array
```

Reply:
`iterable` is a valid return type here, by doing this way we don't create an useless array in memory, and Twig [only iterates on `$extension->getFunctions()`](https://github.com/twigphp/Twig/blob/4a7de4ab5603ad8f199222cae3951fc271a8c45d/src/ExtensionSet.php#L443-L445), so that's perfectly fine.

Note we use `iterable` + `yield` in the repo:
- https://github.com/symfony/ux/blob/80b36bcbeb2902a55495db2afd544f738c73148f/src/Notify/src/Twig/NotifyExtension.php#L25-L28
- https://github.com/symfony/ux/blob/80b36bcbeb2902a55495db2afd544f738c73148f/src/Turbo/src/Twig/TwigExtension.php#L33 

Also, the PHPDoc is useless since `@return TwigFunction[]` is already defined here https://github.com/twigphp/Twig/blob/4a7de4ab5603ad8f199222cae3951fc271a8c45d/src/Extension/ExtensionInterface.php#L58-L63